src/wgpu_graphics_header.rs

```diff
@@ -623,65 +545,6 @@ pub fn setup_render_pass<'a, 'b>(
     rpass
 }
 
-/*
-wgpu::RenderPassDescriptor {
```

Comment:
I don't know what this is referring to or what it's supposed to be but it seems old.

Reply:
See related comment